### PR TITLE
Remove unused offscreen text from psammead-media-indicator

### LINKED ISSUE #1204: media indicator: drop offscreenText
**Is your feature request related to a problem? Please describe.**
We stopped using the offscreenText in the media indicator in bbc/simorgh#2172. We should delete this unused code

**Describe the solution you'd like**
:wave: https://github.com/bbc/psammead/blob/701e4ceb2c79427a129594a3894958925a343b6e/packages/components/psammead-media-indicator/src/index.jsx#L57-L59

**Describe alternatives you've considered**
Keeping the dead code forever 💀 

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #1204 

**Overall change:** _Remove unused media indicator offscreen text._

**Code changes:**

- _Remove unused offscreen text from the MediaIndicator component._
- _Update tests._
- _Update stories._
- _Update snapshots._
---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

**Comment by Bopchy** [comment]
Created https://github.com/bbc/psammead/issues/1242 to update StoryPromo to use this version of MediaIndicator

**Review by andrew-nowak — APPROVED** [review]
deleting dead code :yay:

**Review by ghost — CHANGES_REQUESTED** [review]
Update `version` on `package-lock.json`.

**Review by ghost — CHANGES_REQUESTED** [review]
Now that we don't use the offscreen text in `media-indicator`, we should remove `psammead-visually-hidden-text` from `media-indicator` dependancies.

**Review by ghost — APPROVED** [review]

File: packages/components/psammead-media-indicator/CHANGELOG.md [hunk 1]
```diff
@@ -3,8 +3,9 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 2.1.1 | [PR#1240](https://github.com/bbc/psammead/pull/1240) Remove unused offscreen text |
```
Comment (andrew-nowak) [inline packages/components/psammead-media-indicator/CHANGELOG.md]:
```suggestion
| 3.0.0 | [PR#1240](https://github.com/bbc/psammead/pull/1240) Remove unused offscreen text |
```

File: packages/components/psammead-media-indicator/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-indicator",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (andrew-nowak) [inline packages/components/psammead-media-indicator/package.json]:
```suggestion
  "version": "3.0.0",
```

File: packages/components/psammead-media-indicator/package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbc/psammead-media-indicator",
-  "version": "2.1.0",
+  "version": "2.1.1",
```
Comment (andrew-nowak) [inline packages/components/psammead-media-indicator/package-lock.json]:
```suggestion
  "version": "3.0.0",
```

**Review by andrew-nowak — CHANGES_REQUESTED** [review]
looking again at this, the API is being changed so this is technically a breaking change and should be a major version bump

File: lerna-debug.log [hunk 4]
```diff
@@ -0,0 +1,21 @@
+0 silly argv { _: [ 'exec' ],
```
Comment (dr3) [inline lerna-debug.log]: 😱

**Review by dr3 — COMMENTED** [review]
I think this should have been a minor change, as while the API has changed, its not breaking. Also theres a lerna debug file? TIL theyre a thing 🙈 Other than that, LGTM

**Review by j-pendlebury — APPROVED** [review]
Looks good to me 👍⭐️

**Review by andrew-nowak — APPROVED** [review]

**Comment by david-boydell** [comment]
Cool, this is ready for merge.

**Pull request merged**